mythril/laser/ethereum/cfg.py

```diff
@@ -55,11 +62,7 @@ def __init__(
         self.function_name = function_name
         self.flags = NodeFlags()
 
-        # Self-assign a unique ID
-        global gbl_next_uid
-
-        self.uid = gbl_next_uid
-        gbl_next_uid += 1
+        self.uid = get_new_gbl_id()
```

Comment:
why don't we use the default python id for this?

Reply:
Is there a sequence id function for python?

Reply:
Does the order matter?
Otherwise, every object has an id:
```
a = dict()
id(a)
```